### PR TITLE
WIP - registrationless signature

### PR DESCRIPTION
**Pull request by adridadou**

### CODE REVIEW

File: js/src/APIClient.js [hunk 1]
```diff
@@ -165,6 +165,26 @@ export class APIClient {
       }
     );
   }
+  async prepareSignature(contractId:string, fullName:string, accessToken:?string):Promise<object> {
+      return this.getCall('/signForMetamask/contract/' + contractId,  {
```
Comment (Jdville03) [inline js/src/APIClient.js]: should it be `/prepareSignature/contract/` here? (compared against routes in your related pr in openlaw backend https://github.com/openlawteam/openlaw/pull/1689/files)

File: js/src/APIClient.js [hunk 2]
```diff
@@ -165,6 +165,26 @@ export class APIClient {
...
+          fullName,
+          accessToken
+      });
+  }
+
+  async loadContractStatus(contractId:string, accessToken:?string):Promise<object> {
+      return this.getCall('/contract/sign/status', {
+          id: contractId,
+          accessToken
+      }).then(result => result.data);
```
Comment (Jdville03) [inline js/src/APIClient.js]: in routes in your related pr in openlaw backend https://github.com/openlawteam/openlaw/pull/1689/files i noticed that this GET request has only the `id` parameter. should that be updated to include the accessToken too?

File: js/src/APIClient.js (hunk 1 above)
Comment (adridadou) [reply js/src/APIClient.js]: you're totally right. thanks!

File: js/src/APIClient.js (hunk 2 above)
Comment (adridadou) [reply js/src/APIClient.js]: you're right! fixing it on openlaw

**Pull request merged**